src/AbstractPathResolver.js

```diff
@@ -15,6 +15,16 @@ export default class AbstractPathResolver {
   }
 
   async extendContext(...contexts) {
+    if (contexts.length) {
+      const defaultLanguages = contexts.map(context => context?.['@context']?.['@language']).filter(Boolean);
+
+      if (defaultLanguages.length) {
+        if (!this.defaultLanguage)
+          this.defaultLanguage = defaultLanguages.pop();
+        if (defaultLanguages.some(language => language !== this.defaultLanguage))
+          throw new Error('Multiple default languages given');
+      }
+    }
```

Comment:
I find it weird that we need this check here; can we not make it consistent?
Or just make the last language count?

In any case, if we really need the check, then I would abstract that check into a method `verifyDefaultLanguage` or so.

Reply:
I too kit out and always set it now.